.github/workflows/johnnytest4.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+output_file="CMake_Error_Log.txt"
+
+# Clean the output file first
+echo "" > $output_file
+
+# specify the directory
+directory="./.github/workflows"
```

Comment:
It doesn't work in GitHub action environment. If you want to check where are you in directory tree then use pwd command or see my comment for your changes with smoketest.yaml.